src/pybind/mgr/cephadm/module.py

```diff
@@ -2135,6 +2135,12 @@ def _check_daemons(self):
             # These daemon types require additional configs after creation
             if dd.daemon_type in ['grafana', 'iscsi', 'prometheus', 'alertmanager', 'nfs']:
                 daemons_post[dd.daemon_type].append(dd)
+            
+            if dd.daemon_type in ['mgr', 'mds']:
```

Comment:
@sebastian-philipp The reason I was originally very careful and only allowing mds or mgr daemons here was because I had left `get_active_daemon` undefined for many of the daemon types. I've refactored it a bit so if it's called for a service that hasn't defined the function it just gets back an empty Daemon Desc and then checks if the daemon_id on the Daemon Desc it gets back matches the daemon that is being checked. Thoughts?

Also, the reasoning here is the same reason I was limiting it to mgr and mds elsewhere. I've marked those conversations as resolved but can go back to them if this system is insufficient.

Reply:
lgtm. I'll try to get though QA asap